### DISCUSSION TITLE
Help! Issue with Reusing a Domain in Coolify After Deleting a Project

### DISCUSSION THREAD

**Discussion by estebanrossetti** [coollabsio/coolify#4006] [Q&A]
I'm new to Coolify. I created a WordPress-MySQL project to which I associated a domain, let's say https://adatify.com/. Then I deleted that project. Now I want to create another project similar to that one with the same domain, but I get the following message: "Domain https://adatify.com/ is already in use by another resource: Link: wordpress-with-mysql-y4ks08gokkogw0kg0w04swco."

In Coolify, I go to Servers > Localhost > Resources, and I see that resource listed there, but I cannot delete it.

How can I remove it? Can you help me?

**Comment by sovetski** [comment]
Any update here? I am wondering about this too, if I delete any project or a domain from a project, does it deleted from everywhere? Especially the random domains which are generated by Coolify?
